### ISSUE TITLE
<bug> 

### ISSUE THREAD

**Issue by cdmvirt** [SkytAsul/BeautyQuests#71]
**Reproduction**	
Create quest select NPC Create, try to change NPC type, plugin errors out.

**Expected behavior**
Chosing NPC Type

**Screenshots, error messages**
```
[Server thread/ERROR]: Could not pass event InventoryClickEvent to BeautyQuests v0.15
org.bukkit.event.EventException: null
at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:320) ~[spigot-1.14.jar:git-Spigot-1eece4f-0d76f86]
at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[spigot-1.14.jar:git-Spigot-1eece4f-0d76f86]
at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:520) ~[spigot-1.14.jar:git-Spigot-1eece4f-0d76f86]
at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:505) ~[spigot-1.14.jar:git-Spigot-1eece4f-0d76f86]
at net.minecraft.server.v1_14_R1.PlayerConnection.a(PlayerConnection.java:2158) ~[spigot-1.14.jar:git-Spigot-1eece4f-0d76f86]
at net.minecraft.server.v1_14_R1.PacketPlayInWindowClick.a(SourceFile:33) ~[spigot-1.14.jar:git-Spigot-1eece4f-0d76f86]
at net.minecraft.server.v1_14_R1.PacketPlayInWindowClick.a(SourceFile:10) ~[spigot-1.14.jar:git-Spigot-1eece4f-0d76f86]
at net.minecraft.server.v1_14_R1.PlayerConnectionUtils.lambda$0(PlayerConnectionUtils.java:13) ~[spigot-1.14.jar:git-Spigot-1eece4f-0d76f86]
at net.minecraft.server.v1_14_R1.TickTask.run(SourceFile:18) [spigot-1.14.jar:git-Spigot-1eece4f-0d76f86]
at net.minecraft.server.v1_14_R1.IAsyncTaskHandler.executeTask(SourceFile:135) [spigot-1.14.jar:git-Spigot-1eece4f-0d76f86]
at net.minecraft.server.v1_14_R1.IAsyncTaskHandlerReentrant.executeTask(SourceFile:23) [spigot-1.14.jar:git-Spigot-1eece4f-0d76f86]
at net.minecraft.server.v1_14_R1.IAsyncTaskHandler.executeNext(SourceFile:114) [spigot-1.14.jar:git-Spigot-1eece4f-0d76f86]
at net.minecraft.server.v1_14_R1.MinecraftServer.aW(MinecraftServer.java:896) [spigot-1.14.jar:git-Spigot-1eece4f-0d76f86]
at net.minecraft.server.v1_14_R1.MinecraftServer.executeNext(MinecraftServer.java:889) [spigot-1.14.jar:git-Spigot-1eece4f-0d76f86]
at net.minecraft.server.v1_14_R1.IAsyncTaskHandler.awaitTasks(SourceFile:123) [spigot-1.14.jar:git-Spigot-1eece4f-0d76f86]
at net.minecraft.server.v1_14_R1.MinecraftServer.sleepForTick(MinecraftServer.java:873) [spigot-1.14.jar:git-Spigot-1eece4f-0d76f86]
at net.minecraft.server.v1_14_R1.MinecraftServer.run(MinecraftServer.java:806) [spigot-1.14.jar:git-Spigot-1eece4f-0d76f86]
at java.lang.Thread.run(Thread.java:748) [?:1.8.0_212]
Caused by: java.lang.IllegalArgumentException: Size for custom inventory must be a multiple of 9 between 9 and 54 slots
at org.apache.commons.lang.Validate.isTrue(Validate.java:136) ~[spigot-1.14.jar:git-Spigot-1eece4f-0d76f86]
at org.bukkit.craftbukkit.v1_14_R1.CraftServer.createInventory(CraftServer.java:1545) ~[spigot-1.14.jar:git-Spigot-1eece4f-0d76f86]
at org.bukkit.Bukkit.createInventory(Bukkit.java:1006) ~[spigot-1.14.jar:git-Spigot-1eece4f-0d76f86]
at fr.skytasul.quests.gui.misc.EntityTypeGUI.open(EntityTypeGUI.java:55) ~[?:?]
at fr.skytasul.quests.gui.Inventories.create(Inventories.java:37) ~[?:?]
at fr.skytasul.quests.gui.npc.NPCGUI.onClick(NPCGUI.java:97) ~[?:?]
at fr.skytasul.quests.gui.Inventories.onClick(Inventories.java:60) ~[?:?]
at fr.skytasul.quests.QuestsListener.onClick(QuestsListener.java:90) ~[?:?]
at sun.reflect.GeneratedMethodAccessor656.invoke(Unknown Source) ~[?:?]
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_212]
at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_212]
at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:316) ~[spigot-1.14.jar:git-Spigot-1eece4f-0d76f86]
... 17 more
```

**Softwares**
BeautyQuests version:
Game version: 1.14
Server version: 1.14
Server type (e.g. PaperSpigot): Spigot
Are you using a protocol hack (like ViaVersion)? No

**Comment by Vainock** [comment]
Already fixed, the release needs only to be uploaded to SpigotMC.

**Comment by SkytAsul** [comment]
Fixed on 0.15.1
